lm_eval/models/huggingface.py

```diff
@@ -428,10 +428,9 @@ def _model_generate(
             max_new_tokens=max_tokens,
             stopping_criteria=stopping_criteria,
             do_sample=False,
+            pad_token_id=self.tokenizer.eos_token_id,
         )
-        return utils.select_continuation_from_batch_left_padding(
```

Comment:
Any idea if this change breaks any of the other generation-based tasks like Lambada?